### DISCUSSION TITLE
TimePicker

### DISCUSSION THREAD

**Discussion by tomassev** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2683] [Q&A] [answered]
Hello I would like to create new time picker when I press a button could you help me-> C# XAML

I expect something like TimePicker picker = new TimePicker()
picker.ShowDialog ..........

**Comment by ElieTaillard** [comment, marked as answer]
Create instance:
```
var picker = new TimePicker();
```
Open picker:
```
picker.IsDropDownOpen = true;
```

You can put this code in the button click event.
